### PR TITLE
Spawners

### PR DESCRIPTION
**Pull request by VividPups**
## About the PR
Single Item Loot spawners, Div talk about it before, There is currently a Thread also, However, Due to other PRs merging it is causing a lot of problems with my PRs So I decided to creat them 1 at a time. This being first 
## Why / Balance
Too add RnG to single loot instead of just Static and crates 

## Technical details
N/A

## Media
N/A
- [x] this PR does not require an ingame showcase

## Breaking changes
N/A
**Changelog**
Not really sure if this needs a change log for players. So will leave blank for now

### CODE REVIEW

**Comment by VividPups** [comment]
@dvir001 @MagnusCrowe 

He is the official Pulls. things just keep breaking for me when trying to do stuff, So instead of adding the Bluespace Event with the spawners, Im just going to do Spawners first, then Mob spawners, then the blue space event. to help prevent errors.

**Comment by VividPups** [comment]
As of now, these Commits should be restoring some of the weapons to its OG values, The only thing I haven't done as of now is the Mosin.  I don't know why its like this but it is

**Comment by VividPups** [comment]
ALRIGHT, everything should return back to the Original state now

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/revolvers.yml
Comment (Cheackraze) [inline Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/revolvers.yml:10]: this looks like a typo here, should probably fix

**Review by Cheackraze — COMMENTED** [review]
looks good to me but I want to defer to @MagnusCrowe and @Leander-0 for specific list balancing

**Comment by MagnusCrowe** [comment]
> looks good to me but I want to defer to @MagnusCrowe and @Leander-0 for specific list balancing

Will have a look in the next few days.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/revolvers.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Guns/Guns/revolvers.yml]: Fix it

**Comment by VividPups** [comment]
@MagnusCrowe  Please tell me the line its on if anything needs to be changed there is a lot of lines

**Comment by VividPups** [comment]
is there a reason why there is an Do not merge and need discussion on it?

**Comment by dvir001** [comment]
``DO NOT MERGE`` is added to mark a PR as needing changes and it cannot be added without them.
``Need Discussion`` is added to note it need the changes to be talked about.

Right now im waiting for a maintainer to look at the spawner list, update it and approve when done.

**Comment by VividPups** [comment]
hello, its been about a week, I would like to request a review from yall if yall could

**Comment by MagnusCrowe** [comment]
Hey sorry about that, I'm looking at this now.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:126]: Wait a second it's common and rare...

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:69]: Make rare

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:75]: All these high tech machines have to be removed.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:78]: Probably should be rare but I'd like to hear your opinion.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:92]: Probably should be rare but I'd like to hear your opinion.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:68]: Make rare

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:114]: Remove

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:130]: Rare and common, remove from both

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:176]: No

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:178]: No centcom loot.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:186]: Not rare

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:195]: No admin items.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:197]: Kinda bad loot, doesn't work right last I checked.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:199]: Artifacts aren't really that rare.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:203]: Don't include sec items in a legal list.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:216]: Don't need this line.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:234]: Rare at minimum...

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:242]: Is this illegal?

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:243]: Is this illegal?

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:255]: Not sure how this qualifies as illegal, it has no syndicate markings and can't hold more than the merc variant.  It's really good because of the resistances on it, but this strikes me more as a rare legal item rather than a common illegal item.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:262]: While it says syndicate costume clown in the item ID, it has no contraband inside it as far as I can tell, this can be a common legal loot item.  All it contains is mundane clown gear...not even an agent ID.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:269]: How did this end up here, this isn't illegal.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:271]: Not contraband as far as I can tell.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:283]: Probably rare.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:280]: Is this illegal?  Just seems powerful, make it a rare legal loot

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:282]: Not really illegal...

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:284]: Is there a reason this stuff is even illegal other than it comes in a syndicate bag?  We should change the bag it comes in and let it be legal common.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:288]: Rare if not removed entirely.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:293]: No centcom gear.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:298]: Rare

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:299]: No centcom gear

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:300]: Illegal?

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:304]: Illegal?

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:303]: Illegal?

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:302]: Illegal?

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:325]: Opinions on making this common?  No explosion, just pushes stuff away.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:327]: @dvir001 Reason we didnt want these around?

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:334]: No.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:338]: Why rare, not illegal either.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:339]: Why rare, not illegal either.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:340]: Why rare, not illegal either.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:341]: Not illegal.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:343]: No vending machines.

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Ammunition/explosives.yml
Comment (MagnusCrowe) [inline Resources/Prototypes/_NF/Entities/Objects/Weapons/Ammunition/explosives.yml:39]: What?

**Review by MagnusCrowe — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:234]: image
4 shots to kill a full health death squad in hardsuit.

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:121]: are you wanting all 4 of them removed?

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:255]: thats because I think in upstream the vest web is an syndi item, when I did the illegal spawner I gone through the uplink and a bunch of surplus crates

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (MagnusCrowe) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:121]: Just the decelerator

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:118]: I dont know how it would be good for This to be remove as a chance to get yet, Sec is able to just buy it. I mean when i was sheriff the other day. all the deputys had captain swords

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:135]: Removed all 4 of those

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:159]: Im going to leave this one open. I Remove abunch of the joke ones. but another set of eyes would be better

File: Resources/Prototypes/_NF/Entities/Objects/Weapons/Ammunition/explosives.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Objects/Weapons/Ammunition/explosives.yml:39]: that was because, something mess up and it added things that wasnt post to be added even tho i took it off. So I revert it back to the OG file

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:234]: well thats because it ignores Body armor completely so it does True damage to the person it self

Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:234]: move to Rare

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:269]: its something you can buy in the uplink as an illegal item

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:242]: So I think it was post to be use to talk to borgs after you emag them to give them commands, Either away I remove it

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:300]: Its a pizza that has Omni in it. I was just trying to get filler in it also

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:325]: I mean i could but i was thinking of the effects it would have if you throught it outside and there is people in space... it could be a REAL shitter thing that can happen with it

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:269]: made legal

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:255]: made legal

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:203]: made illegal

File: Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml
Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:243]: change to legal

Comment (VividPups) [reply Resources/Prototypes/_NF/Entities/Markers/Spawners/Random/event.yml:243]: mhmmm and im not quite sure. I dont ever see Civilys with it. and sec can buy it from uplink

**Comment by VividPups** [comment]
that moment when you did all the changes and yet... it didnt push right

**Comment by VividPups** [comment]
Alright, The first Set of Changes have been made. I believe I did everything, I might have miss things since. The one that had all of them got messed up.

**Comment by VividPups** [comment]
what is this post to be?

**Comment by VividPups** [comment]
Hi, I would like Request another review

**Comment by VividPups** [comment]
Due to the lack of communication and response. I'm just going to close this PR request

**Pull request closed without merging**
